<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paint3D</class>
 <widget class="QMainWindow" name="Paint3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paint3D</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSaveAs">
     <property name="title">
      <string>Save As</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionPenColor"/>
    <addaction name="actionPenWidth"/>
    <addaction name="separator"/>
    <addaction name="actionClearScreen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuBrushes">
    <property name="title">
     <string>Brushes</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionMarker"/>
    <addaction name="actionOilBrush"/>
    <addaction name="actionEraser"/>
    <addaction name="actionWaterColorBrush"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
   <addaction name="menuBrushes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPenColor">
   <property name="text">
    <string>Pen Color..</string>
   </property>
  </action>
  <action name="actionPenWidth">
   <property name="text">
    <string>Pen Width..</string>
   </property>
  </action>
  <action name="actionClearScreen">
   <property name="text">
    <string>Clear Screen..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About..</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt..</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionMarker">
   <property name="text">
    <string>Marker</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionOilBrush">
   <property name="text">
    <string>Oil Brush</string>
   </property>
  </action>
  <action name="actionWaterColorBrush">
   <property name="text">
    <string>Watercolor Brush</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/icons/redo.png</normaloff>:/icons/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Resources/resources.qrc">
     <normaloff>:/icons/icons/printer.png</normaloff>:/icons/icons/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
